plugins/woocommerce/client/admin/client/settings-payments/components/modals/woo-payments-post-sandbox-account-setup-modal.tsx

```diff
@@ -72,15 +72,15 @@ export const WooPaymentsPostSandboxAccountSetupModal = ( {
 								<span>.</span>
```

Comment:
Nitpick: we shouldn't break the `<span>` element to insert the `<Link>` - ideally the `<span>` should contain all text content (or, even better, use a `<p>` element if possible). We can also use `interpolateComponents` to interpolate the link so that all the text is translatable, e.g.:

 ```suggestion
  <p>
      { interpolateComponents({
        mixedString: __(
          "We've created a test account for you so that you can begin testing payments on your store. Not sure what to test? Take a look at {{link}}how to test payments{{/link}}.",
          'woocommerce'
        ),
        components: {
          link: (
            <Link
              href="https://woocommerce.com/document/woopayments/testing-and-troubleshooting/sandbox-mode/"
              target="_blank"
              rel="noreferrer"
              type="external"
            />
          ),
        },
      } ) }
</p>
```

Reply:
thanks, will fix it!

Reply:
done in https://github.com/woocommerce/woocommerce/pull/53590/commits/e34e26744fa762872781bd262ad5f219530089cf and https://github.com/woocommerce/woocommerce/pull/53590/commits/06ea5fe011abd6832bd303c07bde94e820efbde7!